File: example/order.go
package example

var order1 = order2 - 10
var (
	order2 = order3 - 10
	order3 = 100
	order4 = order1 - 10
	order5 = order6()
)

func order6() int {
	return 500
}

var _, _ = order4, order5

File: example/order.lua
-- package: example

local gopkg = _G["github.com/edwingeng/go2lua/example"]
do
    local g = _G
    local newEnv = setmetatable({}, {
        __index = function (t, k)
            local v = gopkg[k]
            if v == nil then return g[k] end
            return v
        end,
        __newindex = gopkg,
    })
    _ENV = newEnv
end

order1 = undef
order2 = undef
order3 = undef
order4 = undef
order5 = undef

order6 = function()
    return 500
end

_, _ = undef, undef

return function() end
